yarn.lock

```diff
@@ -21472,6 +21625,22 @@ typedarray@^0.0.6:
   resolved "https://registry.yarnpkg.com/typedarray/-/typedarray-0.0.6.tgz#867ac74e3864187b1d3d47d996a78ec5c8830777"
   integrity sha1-hnrHTjhkGHsdPUfZlqeOxciDB3c=
 
+typescript-plugin-css-modules@^2.2.0:
```

Comment:
are you planning to use this for the scss typing? it doesn't look like it's used anywhere yet in this PR

Reply:
hmm it should be removed.